pushServices/src/hms/java/com/computerrock/push/core/HmsPushService.kt

```diff
@@ -15,4 +27,47 @@ internal class HmsPushService : HmsMessageService() {
         super.onMessageReceived(message)
         message?.let { PushServiceManager.sendNewMessage(HmsRemoteMessage(message)) }
     }
+
+    override fun obtainPushToken(iToken: IToken) {
+        // Create a thread.
+        object : Thread() {
```

Comment:
How will this work when there is more require at the same time?
Is it possible maybe to move this to the coroutines(i just ask, this is not requirement)?

Reply:
Official doc: https://developer.huawei.com/consumer/en/doc/development/HMSCore-Guides/android-client-dev-0000001050042041